src/pages/home/report/ReportActionItem.js

```diff
@@ -328,6 +328,10 @@ function ReportActionItem(props) {
                     isWhisper={isWhisper}
                 />
             );
+        } else if(props.action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && originalMessage && originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY && !isSendingMoney) {
```

Comment:
<img width="1104" alt="Screenshot 2023-10-20 at 6 37 51 PM" src="https://github.com/Expensify/App/assets/35863227/8b81540d-d123-49c2-99e8-1794f4231e61">
<img width="1309" alt="Screenshot 2023-10-20 at 6 37 46 PM" src="https://github.com/Expensify/App/assets/35863227/212e8e4f-48d8-49b7-8700-dc153c6b2b79">


@situchan Any idea what is the best place to add this code because placing them here leaves some spaces.

Reply:
<img width="563" alt="Screenshot 2023-10-20 at 7 19 40 PM" src="https://github.com/Expensify/App/assets/108292595/0af45f82-06ae-48fd-8c44-c5eae6fc3a47">

Reply:
If report action should not be rendered, it doesn't make sense to wrap this null view with all these wrappers - PressableWithSecondaryInteraction, Hoverable, OfflineWithFeedback

Reply:
Wooohhh, nice, that is indeed a good spot. I was trying to figure out outside of return logic.

Reply:
> If report action should not be rendered, it doesn't make sense to wrap this null view with all these wrappers - PressableWithSecondaryInteraction, Hoverable, OfflineWithFeedback

Yeah, exactly.

Reply:
One more question: why not update logic in `ReportActionsUtils.shouldReportActionBeVisible`?

Reply:
I think we haven't discovered the use case of pay action yet, so we will temporarily disable them in the ReportActionItem unlike actions listed in `shouldReportActionBeVisible`, which we will hardly bring back.

Reply:
Ah ok